Makefile

```diff
@@ -32,6 +32,7 @@ $(PYTHON):
 
 html: install
 	$(PELICAN) $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS)
+	cp self-hosting-a-hard-promess.html $(OUTPUTDIR)/en/
```

Comment:
oh, wow, that's a hackety hack (not very scalable)! That solves the current problem though, thanks!